conscious/chain.json

```diff
@@ -0,0 +1,106 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "Conscious",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://www.consciousdao.com/",
+  "pretty_name": "ConsciousDAO",
+  "chain_id": "cvn_2032-1",
+  "bech32_prefix": "cvn",
```

Comment:
Please register this prefix
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
I have registered this prefix.
https://github.com/satoshilabs/slips/pull/1641

Reply:
Marked as draft. Will proceed once merged.

Reply:
OK.